core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -1085,281 +1085,16 @@ oppia.constant('STATE_ERROR_MESSAGES', {
-                    .addClass('state-edit-desc'));
-              }
-              break;
-
-            case 'default_outcome':
-              var defaultOutcomeChanges = getRelativeChangeToGroups(lostChange);
-              var defaultOutcomeHtml = '';
-              if (defaultOutcomeChanges === 'added') {
-                defaultOutcomeHtml += (
-                  '<p class="sub-edit"><i>Destination: </i>' +
-                    newValue.dest + '</p>');
-                defaultOutcomeHtml += (
-                  '<div class="sub-edit"><i>Feedback: </i>' +
-                    '<div class="feedback">' + newValue.feedback.getHtml() +
-                    '</div></div>');
-                stateWiseEditsMapping[stateName].push(
-                  angular.element('<div>Added default outcome: </div>')
-                    .append(defaultOutcomeHtml)
-                    .addClass('state-edit-desc default-outcome'));
-              } else if (defaultOutcomeChanges === 'edited') {
-                if (newValue.dest !== oldValue.dest) {
-                  defaultOutcomeHtml += (
-                    '<p class="sub-edit"><i>Destination: </i>' + newValue.dest +
-                      '</p>');
-                }
-                if (!angular.equals(newValue.feedback.getHtml(),
-                  oldValue.feedback.getHtml())) {
-                  defaultOutcomeHtml += (
-                    '<div class="sub-edit"><i>Feedback: </i>' +
-                      '<div class="feedback">' + newValue.feedback +
-                      '</div></div>');
-                }
-                stateWiseEditsMapping[stateName].push(
-                  angular.element('<div>Edited default outcome: </div>')
-                    .append(defaultOutcomeHtml)
-                    .addClass('state-edit-desc default-outcome'));
-              } else if (defaultOutcomeChanges === 'deleted') {
-                stateWiseEditsMapping[stateName].push(
-                  angular.element('<div>Deleted default outcome</div>')
-                    .addClass('state-edit-desc'));
-              }
-          }
-      }
-    });
-
-    for (var stateName in stateWiseEditsMapping) {
-      var stateChangesEl = angular.element(
-        '<li>Edits to state: ' + stateName + '</li>');
-      for (var stateEdit in stateWiseEditsMapping[stateName]) {
-        stateChangesEl.append(stateWiseEditsMapping[stateName][stateEdit]);
-      }
-      outerHtml.append(stateChangesEl);
-    }
-
-    return outerHtml;
-  };
-
-  return {
-    makeHumanReadable: makeHumanReadable
-  };
-}]);
-
 // Service for displaying different types of modals depending on the type of
 // response received as a result of the autosaving request.
 oppia.factory('autosaveInfoModalsService', [
-  '$log', '$uibModal', '$timeout', '$window',
-  'ExplorationDataService', 'LocalStorageService', 'lostChangesService',
-  'UrlInterpolationService',
+  '$log', '$modal', '$timeout', '$window',
```

Comment:
Take a look here. You haven't resolved to merge conflict correctly (I guess). Change the $modal to $uibModal. Also, you should take a final look at all the changes while making a PR from github. I hope this may help you.